### PR TITLE
Rectify functional tests and lint failure

### PR DESCRIPTION
**Pull request by sinhaashish**

### CODE REVIEW

File: pkg/uevent/indexer_test.go [hunk 1]
```diff
@@ -64,11 +64,17 @@ func TestFilterDrivesByUEventFSUUID(t *testing.T) {
 
 	d1 := createTestDrive("test-node", "D1", "d9877501-e1b5-4bac-b73f-178b29974ed5")
 	d2 := createTestDrive("test-node", "D2", filterOnFSuuid)
-
+	var err error
 	indexer := createFakeIndexer()
-	indexer.store.Add(d1)
-	indexer.store.Add(d2)
-	filteredDrive, _ := indexer.filterDrivesByUEventFSUUID(filterOnFSuuid)
+	_ = indexer.store.Add(d1)
```
Comment (Praveenrajmani) [inline pkg/uevent/indexer_test.go]: can we not ignore the error here?

File: pkg/uevent/indexer_test.go [hunk 2]
```diff
@@ -80,10 +86,14 @@ func TestListDrives(t *testing.T) {
 	d1 := createTestDrive("test-node", "D1", "d9877501-e1b5-4bac-b73f-178b29974ed5")
 	d2 := createTestDrive("test-node", "D2", "b9475609-e1b5-4986-vs33-178131rdes9")
 	var drives []*directcsi.DirectCSIDrive
+	var err error
 	drives = append(drives, d1, d2)
 	indexer := createFakeIndexer()
-	indexer.store.Add(d1)
-	indexer.store.Add(d2)
+	_ = indexer.store.Add(d1)
```
Comment (Praveenrajmani) [inline pkg/uevent/indexer_test.go]: can we not ignore the error here?

File: pkg/uevent/indexer_test.go (hunk 1 above)
Comment (sinhaashish) [reply pkg/uevent/indexer_test.go]: we can , but `golint ./... ` was complaining so i added the same .

File: pkg/uevent/indexer_test.go (hunk 2 above)
Comment (sinhaashish) [reply pkg/uevent/indexer_test.go]: we can , but `golint ./... ` was complaining so i added the same .

Comment (Praveenrajmani) [reply pkg/uevent/indexer_test.go]: can you try 

```
if err := indexer.store.Add(d2); err != nil {
            t.Errorf("error while adding objects to store: %v", err)
}
```

File: pkg/uevent/indexer_test.go [hunk 3]
```diff
@@ -64,11 +65,18 @@ func TestFilterDrivesByUEventFSUUID(t *testing.T) {
 
 	d1 := createTestDrive("test-node", "D1", "d9877501-e1b5-4bac-b73f-178b29974ed5")
 	d2 := createTestDrive("test-node", "D2", filterOnFSuuid)
-
+	var err error
```
Comment (Praveenrajmani) [inline pkg/uevent/indexer_test.go]: remove this var declaration if it is not used @sinhaashish

**Pull request merged**